capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
```

Comment:
### Optional
- [ ] It would be nice if we use `require_relative` when working with files in our current directory, we can use `require` when working with packages in our Gemfile

Reply:
Noted. Thanks 👍